src/content/blog/astro-5130.mdx

```diff
@@ -0,0 +1,202 @@
+- [**Experimental Chrome DevTools workspace support**](#experimental-chrome-devtools-workspace-support)
+- [**Multiple sitemaps**](#multiple-sitemaps)
+- [**Experimental hosted error pages with the Node.js adapter**](#experimental-hosted-error-pages)
+- [**Enum support in Astro DB tables**](#enum-support-in-astro-db-tables)
+- [**Local support for Cloudflare KV**](#local-support-for-cloudflare-kv)
+
+To upgrade an existing project, use the automated `@astrojs/upgrade` CLI tool. Alternatively, upgrade manually by running the upgrade command for your package manager:
+
+```sh
+# Recommended:
+npx @astrojs/upgrade
+
+# Manual:
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Experimental static `import.meta.env`
+
+Environment variables can be complicated! Astro's `astro:env` API allows you to configure a [type-safe schema for your environment variables](https://docs.astro.build/en/guides/environment-variables/#type-safe-environment-variables), and converts variables imported via `astro:env` into the expected type. This is the recommended way to use environment variables in Astro, as it allows you to easily see and manage whether your variables are public or secret, available on the client or only on the server at build time, and the data type of your values.
+
+However, you can still access your environment variables through `process.env` as well as `import.meta.env` directly if needed. This was the only way to use environment variables in Astro before `astro:env` was added in Astro 5.0, and its handling of `import.meta.env` includes some logic that was intended for earlier versions of Astro that is no longer necessary.
+
+Astro 5.13 includes a new `experimental.staticImportMetaEnv` flag that updates the behavior of `import.meta.env` to align with [Vite's handling of environment variables](https://vite.dev/guide/env-and-mode.html#env-variables) and for better ease of use with Astro's current implementations and features. This flag supersedes the previous `rawEnvVariables` as a more complete solution for handling private meta environment variables. **This will become the default behavior in Astro 6.0**, and this early preview is introduced as an experimental feature. 
+
+Currently, non-public `import.meta.env` environment variables are replaced by a reference to `process.env`. Additionally, Astro may also convert the value type of your environment variables used through `import.meta.env`, which can prevent access to some values such as the strings `"true"` (which is converted to a boolean value), and `"1"` (which is converted to a number).
+
+The `experimental.staticImportMetaEnv` flag simplifies Astro's default behavior, making it easier to understand and use. Astro will no longer replace any `import.meta.env` environment variables with a `process.env` call, nor will it coerce values. Instead, your private meta environment variables will always be inlined.
```

Comment:
@florian-lefebvre Please review the text of this whole section, but in particular I've added a line here that wasn't in any earlier documentation that I want to make sure is said accurately!

> Instead, your private meta environment variables will always be inlined.

I wanted to introduce the language we're using for the title of the feature guide by saying this in "plain language"  so I used those terms here!

Reply:
The whole section LGTM!